src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -13,7 +13,11 @@
 # limitations under the License.
 """Helper functions for app-specific trials/experiments."""
 
+import json
+import os
 import random
+from warnings import warn
```

Comment:
Please use our `logs.log_warn` function.

e.g. https://github.com/google/clusterfuzz/blob/4f8020c4c7ce73c1da0e68f04943af30bb5f0b32/src/clusterfuzz/_internal/platforms/android/adb.py#L145

You'll need to import `from clusterfuzz._internal.metrics import logs`

Reply:
done